apps/condo/domains/billing/schema/BillingAccount.js

```diff
@@ -89,6 +91,15 @@ const BillingAccount = new GQLListSchema('BillingAccount', {
             }
         },
     },
+    kmigratorOptions: {
+        constraints: [
+            {
+                type: 'models.UniqueConstraint',
+                fields: ['context', 'globalId'],
+                name: 'billingAccount_unique_context_globalId',
```

Comment:
Name prefix `billingAccount` is redundant, because constraints are scoped to table in DB.

```suggestion
                name: 'unique_context_globalId',
```

Reply:
For some reason, django is not able to generate migrations when constraint names are not unique among models:

![image](https://user-images.githubusercontent.com/33755274/134657998-944f12ce-e99e-4095-99b2-86054096a110.png)

https://docs.djangoproject.com/en/3.2/ref/checks/

> Models.E302

Reply:
Hmm, it's a redundant restriction, because on database level constraints are enforced to have unique names only in scope of table. Ok.